js/forum/src/components/AvatarEditor.js

```diff
@@ -37,10 +37,10 @@ export default class AvatarEditor extends Component {
     return (
       <div className={'AvatarEditor Dropdown ' + this.props.className + (this.loading ? ' loading' : '')}>
         {avatar(user)}
-        <a className="Dropdown-toggle"
-          data-toggle="dropdown"
-          onclick={this.quickUpload.bind(this)}>
-          {this.loading ? LoadingIndicator.component() : icon('pencil')}
+        <a className={ user.avatarUrl() ? "Dropdown-toggle" : "Dropdown-toggle AvatarEditor--noAvatar" }
+           data-toggle="dropdown"
+           onclick={this.quickUpload.bind(this)}>
```

Comment:
I don't know, I probably just wanted it so I could read it better just in case it didn't work :stuck_out_tongue_winking_eye: